docs/coding-guidelines/adding-api-guidelines.md

```diff
@@ -58,6 +59,17 @@ the implementation without compat concerns in future releases.
   - Add new test code following [conventions](project-guidelines.md#code-file-naming-conventions) for new files to that are specific to the new target framework.
   - To run just the new test configuration run `dotnet msbuild <Library>.csproj /t:RebuildAndTest /p:TargetGroup=<TargetGroup>`
 
+## Documentation
+
+New public APIs must be documented with triple slash comments on top of them. Visual Studio automatically generates the structure for you when you type `\\\`.
+
+If your new API or the APIs it calls throw any exceptions, those need to be manually documented by adding the `<exception></exception>` elements.
+
+After your change is merged, we will eventually port them to the dotnet-api-docs repo, where we will review them for language and proper style (See the [Docs contributing guide](https://github.com/dotnet/docs/blob/master/CONTRIBUTING.md) for more information).
```

Comment:
It would probably be best to link to https://github.com/dotnet/dotnet-api-docs/wiki instead.

```suggestion
After your change is merged, we will eventually port them to the dotnet-api-docs repo, where we will review them for language and proper style (For more information, see the [API writing guidelines](https://github.com/dotnet/dotnet-api-docs/wiki)).
```